<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-Q1BOVCQ" failures="1" tests="2" name="Test" time="0.068" errors="0" timestamp="2024-12-16T08:23:25 IST">
  <testcase classname="tests.XMLParameterisedTests" name="fetchData" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1">
      <![CDATA[java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
at tests.XMLParameterisedTests.getData(XMLParameterisedTests.java:30)
... 25 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- fetchData -->
  <testcase classname="tests.XMLParameterisedTests" name="getURL" time="0.004"/>
</testsuite> <!-- Test -->
